### PR TITLE
2.0 updates

### PR DESCRIPTION
**Pull request by evilnick**

### CODE REVIEW

File: src/en/charms-constraints.md [hunk 1]
```diff
@@ -1,132 +1,173 @@
...
+  value is mega-/giga-/tera-/peta- bytes.
+
+  - **cpu-cores** :  How many cpu-cores the host machine should have. This is a
+  crude indicator of system performance.
+    
+  - **spaces** : Target a particular network space, or avoid one (not supported
+  all clouds).
+  
+  - **arch** : Short for 'architecture', indicates the processor type a service 
+  must run on. One of amd64, arm, i386, arm64, or ppc64.
```
Comment (mbruzek) [inline src/en/charms-constraints.md]: Is it ppc64 or ppc64le? @kwmonroe What do you use on power hardware?

**Comment by mbruzek** [comment]
@evilnick I don't know the proper identifier for Power hardware, I thought it was ppc64le.

File: src/en/charms-constraints.md (hunk 1 above)
Comment (marcoceppi) [inline src/en/charms-constraints.md]: ppc64el

**Comment by evilnick** [comment]
it *is* ppc64el. ISTR there was some reason ppc64 exists too - something to do with older series, but I'm not sure that matters now

**Comment by evilnick** [comment]
@pmatulis ?

**Comment by pmatulis** [comment]
These differ in terms of endian modes.
ppc64el/ppc64le : PowerPC 64-bit little-endian
powerpc/ppc64 : IBM/Motorola PowerPC big-endian

**Comment by evilnick** [comment]
are both still supported though?

**Comment by pmatulis** [comment]
They are supported architectures (since Trusty) if that's what you mean.

**Pull request merged**